project.clj

```diff
@@ -63,6 +64,7 @@
                  [puppetlabs/trapperkeeper-authorization]
                  [puppetlabs/trapperkeeper-scheduler]
                  [puppetlabs/trapperkeeper-status]
+                 [puppetlabs/trapperkeeper-filesystem-watcher "1.0.1"]
```

Comment:
tk-filesystem-watcher is now being managed with clj-parent, as of version 0.1.9, but it was not as of the version that puppetserver 2.x is currently using, 0.1.7.  It may be best to do another pre-1.0.0 release of clj-parent which bumps to the new tk-jetty9 version (when available) and uses the tk-filesystem-watcher version as well.